adminSiteClient/imagesHelpers.ts

```diff
@@ -57,14 +65,69 @@ export async function reuploadImageFromSourceUrl({
     payload.filename = image.filename
 
     const response = await admin.requestJSON<ImageUploadResponse>(
-        `/api/images/${image.id}`,
+        image.id ? `/api/images/${image.id}` : `/api/images`,
         payload,
-        "PUT"
+        image.id ? "PUT" : "POST"
     )
 
     return response
 }
 
+export async function fetchFigmaProvidedImageUrl(
+    figmaUrl: string
+): Promise<FigmaResponse> {
+    const figmaApi = new Figma.Api({
+        personalAccessToken: FIGMA_API_KEY,
```

Comment:
Just noticing that I _didn't_ have this set, because it's not in the `.env.example-full` - we should put it there and mark it as optional 🙂